### DISCUSSION TITLE
Is there any similar solution to qml to react on property change?

### DISCUSSION THREAD

**Discussion by qarmin** [slint-ui/slint#3800] [Q&A]
e.g. in qml if property
```
property int width: 20
```
will be changed
```
width = 30;
```
this function will be executed 
```
onWidthChanged(width): {
    console.error("changed width to:", width)
}
 ```

 Is there any similar solution in slint, or I need to use callback `set_width` if I want to get similar functionality?

**Comment by bjorn** [comment]
Such a callback does not exist automatically yet, so indeed for now you'd have to add an explicit one. See issue #112 about this feature.
